CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# dbt-utils v0.7.4
+🚨 This is a compatibility release in preparation for `dbt-core` v1.0.0 (🎉). Projects using dbt-utils 0.7.4 with dbt-core v1.0.0 can expect to see a deprecation warning. This will be resolved in dbt_utils v0.8.0.
+
+## Fixes
+- `get_column_values()` now works correctly with mixed-quoting styles on Snowflake ([#424](https://github.com/dbt-labs/dbt-utils/issues/424), [#440](https://github.com/dbt-labs/dbt-utils/pull/440))
```

Comment:
this is nitpicky but should we say what we did / changed to make it work with mixed-quoting styles on Snowflake?

Reply:
I think release notes can be the what; people can click through to the PR for the how